crates/katana/rpc/rpc/src/dev.rs

```diff
@@ -92,6 +97,35 @@ impl<EF: ExecutorFactory> DevApiServer for DevApi<EF> {
         Ok(())
     }
 
+    async fn account_balance(&self, address: String) -> Result<u128, Error> {
+        let account_address: ContractAddress = Felt::from_str(&address).unwrap().into();
+        let provider = self.backend.blockchain.provider();
+        let state = provider.latest().unwrap();
+        // let storage_slot =
+        //     get_storage_var_address("ERC20_balances", &[account_address.into()]).unwrap();
+        let storage_slot = get_fee_token_balance_base_storage_address(account_address);
+        let balance_felt = state
+            .storage(self.backend.chain_spec.fee_contracts.eth, storage_slot)
+            .unwrap()
+            .unwrap();
+        let balance: u128 = balance_felt.to_string().parse().unwrap();
+        Ok(balance)
+    }
+
+    async fn fee_token(&self) -> Result<String, Error> {
+        let provider = self.backend.blockchain.provider();
+        let state = provider.latest().unwrap();
+        let fee_token = state
+            .storage(self.backend.chain_spec.fee_contracts.eth, ERC20_NAME_STORAGE_SLOT)
+            .unwrap()
+            .unwrap();
+        Ok(fee_token.to_string())
+    }
```

Comment:
since #2536, by default we now have ETH and STRK fee tokens. for v1 and v3 txs respectively. im not exactly sure how we should handle this bcs you can't configure to use only a certain fee token for all tx versions.

i dont know what's the historical context for this method, but `starknet-devnet` seems to just return an error for this.

https://github.com/0xSpaceShard/starknet-devnet-rs/blob/e3c69015fa19f261487ea094caee2c72cdc3a64b/crates/starknet-devnet-server/src/api/http/endpoints/mint_token.rs#L15

lets just remove this one.